09-features.md

```diff
@@ -41,6 +41,7 @@ The Context column decodes as follows:
 | 20/21 | `option_anchor_outputs`          | Anchor outputs                                            | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)         |
 | 22/23 | `option_anchors_zero_fee_htlc_tx` | Anchor commitment type with zero fee HTLC transactions   | IN       | `option_static_remotekey` | [BOLT #3][bolt03-htlc-tx], [lightning-dev][ml-sighash-single-harmful]|
 | 26/27 | `option_shutdown_anysegwit`         | Future segwit versions allowed in `shutdown`              | IN       |                   | [BOLT #2][bolt02-shutdown]   |
+| 28/29 | `option_fat_error`               | Can generate/relay fat errors in `update_fail_htlc`       | IN9      |                   | [BOLT #4][bolt04-fat-errors]   |
```

Comment:
nobody likes a naming nitpick, but something like `rich_error` or `extendable_error` may be a _slightly_ more computer-science-ey term

Reply:
good point. how about `option_attributable_error`?

Reply:
sgtm!